core/web/presenters/eth_key.go

```diff
@@ -5,22 +5,21 @@ import (
 
 	"github.com/smartcontractkit/chainlink/core/assets"
 	"github.com/smartcontractkit/chainlink/core/store/models"
-	"gorm.io/gorm"
 )
 
 // ETHKeyResource represents a ETH key JSONAPI resource. It holds the hex
 // representation of the address plus it's ETH & LINK balances
 type ETHKeyResource struct {
 	JAID
-	Address     string         `json:"address"`
-	EthBalance  *assets.Eth    `json:"ethBalance"`
-	LinkBalance *assets.Link   `json:"linkBalance"`
-	NextNonce   int64          `json:"nextNonce"`
-	LastUsed    *time.Time     `json:"lastUsed"`
-	IsFunding   bool           `json:"isFunding"`
-	CreatedAt   time.Time      `json:"createdAt"`
-	UpdatedAt   time.Time      `json:"updatedAt"`
-	DeletedAt   gorm.DeletedAt `json:"deletedAt"`
+	Address     string       `json:"address"`
+	EthBalance  *assets.Eth  `json:"ethBalance"`
+	LinkBalance *assets.Link `json:"linkBalance"`
+	NextNonce   int64        `json:"nextNonce"`
+	LastUsed    *time.Time   `json:"lastUsed"`
+	IsFunding   bool         `json:"isFunding"`
+	CreatedAt   time.Time    `json:"createdAt"`
+	UpdatedAt   time.Time    `json:"updatedAt"`
+	DeletedAt   *time.Time   `json:"deletedAt"`
```

Comment:
Why this change?

Reply:
I don't want the presenters to have a dependency on `gorm`. It's a DB level concern so it shouldn't leak to the presentational layer